phpstan.neon

```diff
@@ -1,16 +1,35 @@
+includes:
+	- phpstan-baseline.neon
+
 parameters:
-    level: 1
+    level: max
+
     checkFunctionNameCase: true
-    checkMissingIterableValueType: true
     checkInternalClassCaseSensitivity: true
-    checkDynamicProperties: true
+    treatPhpDocTypesAsCertain: false
+
     paths:
         - application
         - library
+
     scanDirectories:
         - vendor
+
     ignoreErrors:
-        - '#Unsafe usage of new static\(\)#'
+        -
+            messages:
+                - '#Unsafe usage of new static\(\)#'
+                - '#. but return statement is missing#'
+            reportUnmatched: false
+
+        - '#Call to an undefined method Icinga\\Module\\Cube\\Ido\\ZfSelectWrapper::joinLeft\(\)#'
+
+        - '#Call to an undefined method Icinga\\Data\\SimpleQuery::.*#'
+
+        - '#Call to an undefined method Icinga\\Data\\ConnectionInterface::getResource\(\)#'
+
+        - '#Call to an undefined method Icinga\\Module\\Cube\\Ido\\IdoCube::addDimensionByName\(\)#'
```

Comment:
I think these can also be included in the baseline. If the bug pops up later, we want to fix it.

Reply:
Why? These are just some interface method & magic method call errors that don't need to be fixed.